### PR TITLE
GUVNOR-3003: [Library] [Assets list] Order assets alphabetically

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/GUVNOR-3003

### CODE REVIEW

**Comment by manstis** [comment]
Part of an ensemble:

- https://github.com/AppFormer/uberfire/pull/697
- https://github.com/kiegroup/kie-wb-common/pull/866
- https://github.com/kiegroup/drools-wb/pull/493
- https://github.com/kiegroup/jbpm-designer/pull/595
- https://github.com/kiegroup/jbpm-form-modeler/pull/105
- https://github.com/kiegroup/optaplanner-wb/pull/163

**Comment by manstis** [comment]
Jenkins please retest this.

File: jbpm-form-modeler-panels/jbpm-form-modeler-editor/jbpm-form-modeler-editor-backend/src/main/java/org/jbpm/formModeler/panels/modeler/backend/indexing/FormIndexVisitor.java [hunk 1]
```diff
@@ -21,9 +21,9 @@
 import org.jbpm.formModeler.api.model.Field;
 import org.jbpm.formModeler.api.model.FieldType;
 import org.jbpm.formModeler.api.model.Form;
+import org.kie.workbench.common.services.refactoring.Resource;
+import org.kie.workbench.common.services.refactoring.ResourceReference;
```
Comment (jhrcek) [inline jbpm-form-modeler-panels/jbpm-form-modeler-editor/jbpm-form-modeler-editor-backend/src/main/java/org/jbpm/formModeler/panels/modeler/backend/indexing/FormIndexVisitor.java]: @manstis Could you please explain for us uninitiated what's the difference between the two and when should the first one be used over the other?

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
@pefernan OK with you?

**Pull request merged**